rrat/tailwind.config.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
+  // Enabling JIT mode for faster builds and more flexibility in using dynamic class names
+  mode: 'jit',
```

Comment:
I don't think we need Just In Time. Comment out/delete for now, we can always add back later.